daml_dit_if/api/__init__.py

```diff
@@ -3,17 +3,19 @@
 
 from typing import Any, Callable, Optional, Sequence
 
-
 from aiohttp.web import Response
 
 from dazl import Command, ContractData, ContractId
 from dazl.model.core import ContractMatch
```

Comment:
I thought this would have emitted a warning, but this import should change to `dazl.query` instead.

Reply:
No warning. Resolved.